extra/gitdm/mailmap

```diff
@@ -7,3 +7,4 @@ Dominik Adamski <adamski.dominik@gmail.com>
 Alex Miliukov <28045858+0lmi@users.noreply.github.com>
 Lluis Campos <lluis.campos@northern.tech>
 Ossian Riday <Ossian.Riday@gmail.com>
+Nils Olav Kvelvane Johansen <nils.olav@northern.tech>
```

Comment:
@NilsOlavKJohansen Is this how you want your name spelled?

Also, please configure your git (see [here](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup), for example) so that your commits get signed with your name correctly spelled.

Reply:
Yes, just like that. 
Thanks for the tip, I'll look into it now.